src/OrchardCore.Modules/OrchardCore.Contents/Drivers/ContentsDriver.cs

```diff
@@ -34,19 +34,17 @@ public override async Task<IDisplayResult> DisplayAsync(ContentItem contentItem,
             // We add custom alternates. This could be done generically to all shapes coming from ContentDisplayDriver but right now it's
             // only necessary on this shape. Otherwise c.f. ContentPartDisplayDriver
 
-            var contentItemViewModel = new ContentItemViewModel(contentItem);
-
             var results = new List<IDisplayResult>()
             {
-                Shape("ContentsTags_SummaryAdmin", contentItemViewModel).Location("SummaryAdmin", "Tags:10"),
-                Shape("ContentsMeta_SummaryAdmin", contentItemViewModel).Location("SummaryAdmin", "Meta:20"),
+                Shape("ContentsTags_SummaryAdmin", new ContentItemViewModel(contentItem)).Location("SummaryAdmin", "Tags:10"),
+                Shape("ContentsMeta_SummaryAdmin", new ContentItemViewModel(contentItem)).Location("SummaryAdmin", "Meta:20"),
```

Comment:
Why you need to new up `ContentItemViewModel` in each call?!!

Reply:
I am not positive on this response. But, I believe because the VM here is a shape but rendered using a different type. For it to work correctly, each shape should be a new instance as it'll have to be built and rendered separately. 

The PR #15652 caused the issue which created a single VM instead of having multiple

Reply:
Seems weird

Reply:
Yeah. It's worth trying to pass a model instead of a shape to all these views. In other words, do not let ContentItemViewModel inherit from ShapeViewModel, create a single instance and pass it to all these views. This may work, but could break someone. Anyway, if it works it should reduce allocations. Feel free to try it. Otherwise, I may try it when I have time. Meanwhile this PR fixes the referenced problem while we can entertain other ways to improve.